climate-monitoring-server/src/main/java/climatemonitoring/Skeleton.java

```diff
@@ -500,6 +500,25 @@ public void run() {
 						break;
 					}
 
+					case INCLUDE_AREA_TO_CENTER: {
+
+						int geonameID = (Integer) m_in.readObject();
+						String centerID = (String) m_in.readObject();
+
+						try {
+
+							m_serverDatabase.includeAreaToCenter(geonameID, centerID);
+
+							m_out.writeObject(true);
+						}
+
+						catch (DatabaseRequestException e) {
+	
+							m_out.writeObject(false);
+							m_out.writeObject(e);
+						}
+					}
+
 					case MONITORS: {
```

Comment:
You are missing a `break`

Reply:
![](https://i.kym-cdn.com/entries/icons/original/000/038/646/E_HXiZqX0Ac2UyZ.jpg)

Reply:
79.50.205.224